src/wasm/wat-parser-internal.h

```diff
@@ -441,6 +511,80 @@ std::optional<LexResult> ident(std::string_view in) {
   return {};
 }
 
+// string     ::= '"' (b*:stringelem)* '"'  => concat((b*)*)
+//                    (if |concat((b*)*)| < 2^32)
+// stringelem ::= c:stringchar              => utf8(c)
+//              | '\' n:hexdigit m:hexdigit => 16*n + m
```

Comment:
What is this rule?

Reply:
This lets you put things like `\00\ff` in your strings to encode byte values (here `00 ff`) that don't have corresponding printable characters.